Lib/test/test_json/test_speedups.py

```diff
@@ -31,6 +31,8 @@ def test(value):
 
 class TestEncode(CTest):
     def test_make_encoder(self):
+        # Issue #6986: The interpreter shouldn't crash in case c_make_encoder()
```

Comment:
Please use the new syntax "bpo-6986" rather than "Issue #6986", it prevents confusion between bugs.python.org bug tracker and GitHub bug tracker ;-)